jobbergate-api/Makefile

```diff
@@ -34,17 +34,13 @@ format: install
 release-%:
 	../release-scripts/prepare-release.sh $(subst release-,,$@)
 
-.PHONY: run
-run:
-	poetry run uvicorn --host 0.0.0.0 jobbergate_api.main:app --reload
-
 .PHONY: publish
 publish:
 	../release-scripts/publish-release.sh
 
 .PHONY: local
 local: install
-	poetry run uvicorn --host 0.0.0.0 jobbergate_api.main:app --reload
+	poetry run jobbergate dev-server
```

Comment:
I'm not sure we want to add dev tools to the `jobbergate` cli. Is there a better place to put the `dev-server` functionality? Or possibly there is a reason you put it here?